rdr_service/resource/helpers.py

```diff
@@ -109,3 +110,23 @@ def get_intersection(self, other_collection):
 
     def any(self):
         return len(self.ranges) > 0
+
+
+def _import_rural_zipcodes():
+    """
+    Load the file app_data/rural_zipcodes.txt and return a list of rural zipcodes.
+    """
+    paths = ('app_data', 'rdr_service/app_data', 'rest-api/app_data')
+    codes = list()
+
+    for path in paths:
+        if os.path.exists(os.path.join(path, 'rural_zipcodes.txt')):
+            with open(os.path.join(path, 'rural_zipcodes.txt')) as handle:
+                # pylint: disable=unused-variable
+                for count, line in enumerate(handle):
```

Comment:
I don't remember why I used 'enumerate' there, but it works fine as is.

Reply:
It works, but do you agree that it's less readable and affects maintainability? I know it's small, but I'm hoping we can have more discussions as a team and learn from each other.

Reply:
Kenny, look I don't want to be insensitive to your concerns here and I certainly don't believe this code is the most Pythonic way to enumerate through the file line by line.  But I do believe that the code functions correctly and is readable and anyone with Python knowledge will understand exactly what it is doing.

This line of code was written in 2019 and has functioned correctly and without errors since then.  On a scale of 1 to 10, my priority on updating this code to be the way you want it to look is -100. Peggy and I are trying to accomplish multiple big tasks in PDR under a time crunch, this is just not a priority for me right now.  I hope that you will understand that I would rather meet deadlines with working code than deliver pretty code late.

But if it is bothering you so badly please change it and create a PR, I will be happy to approve it. Otherwise it can wait until the RDR 2.0 rewrite.